.ci/util.shlib

```diff
@@ -249,3 +249,12 @@ function UTIL_READ_CONFIG_FILE() {
         fi
     done
 }
+
+function UTIL_LOAD_CUSTOM_HOOK() {
+	local hook_name="$1"
+	local hook_file="./${package}/.ci/${hook_name}.sh"
```

Comment:
This should be loading a file directly, not convert a package name into a file given the function name